tools/nxfuse/README_SMARTFS.md

```diff
@@ -1,20 +1,85 @@
-nxfuse
-======
+# How to make smartfs user image for TizenRT
+
+## Contents
+> [How to Enable SmartFS user image](#how-to-enable-smartfs-user-image)  
+> [About NxFuse tool](#about-nxfuse-tool)  
+> [How to use nxfuse](#how-to-use-nxfuse)  
+> [Formatting and Reformatting](#formatting-and-reformatting)  
+
+## How to Enable SmartFS user image
+SMARTFS file system(CONFIG_FS_SMARTFS) needs to be enabled in order to use SMARTFS image.
+
+Enable FS_SMARTFS config through menuconfig at *os* folder.
+
+### Configurations
+Enable below configurations as followings
+
+```
+->BuildSetup
+	->Binary Output Formats
+		-> Enable "Make smartfs binary image" (=y)
+```
+
+```
+->Board Selection
+	-> Adjust smartfs partition size
+```
+
+```
+->File Systems
+	-> Memory Technology Device (MTD) Support
+		-> SMART Device options
+			->  set "SMART Device sector size" = (512 bytes)
+```
+
+```
+->File Systems
+	-> Memory Technology Device (MTD) Support
+		-> SMART Device options
+			->  Enable "Enable Sector CRC error detection" (=y)
+```
+
+### SmartFS Partition size
+Adjust blkcount in $TIZENRT_BASEDIR/tools/nxfuse/mksmartfsimg.sh. blkcount multiplied by blksize should be equal to SMARTFS partition size.
```

Comment:
Could you guide how to check the SAMRTFS partition size? Additionally is it possible to get the value from .config automatically?

Reply:
step1: make menuconfig
step2: Board Selection  --->
	(2048,256,256,256,512,512,**### 2048**,) Flash partition size list (in KBytes)
	(kernel,none,bin,bin,bin,bin,**### smartfs**,) Flash partition type list
	(kernel,app,micom,micom,wifi,wifi,**### userfs**,) Flash partition name list

Reply:
1. I mean you should add the way of checking smartfs partition size here. (Not for me)
2. You already guide setting smartfs partition size at second configuration.
```
->Board Selection
	-> Adjust smartfs partition size
```
Without partition table, it is impossible to download smartfs image into flash.
I think if there are configs for partition size, blk size, page size and so on in .config, we had better use it first. If there is no configs for them, then we can request giving them.

Reply:
2nd could be treated at another PR. That's not mandatory for this PR. I just want to know your opinion.
The reasons why I think are
1. Requested data like above are mandatory to work fs or system.
2. It is not easy to use for all. I don't know what blkcount, blksize, pagesize and erase size are.

Reply:
@sunghan-chang: 
we have blk-size info in .config and we used directly. 
For partition-size, we guided how to fetch from .config
erase-size & page-size we need to depend on vendor inputs only. But from now on we can create these variable also in .config for all upcoming products.